plugin/trino-ranger/src/test/resources/logback.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Why do we use logback? We should follow the same logging method as elsewhere in Trino. See `io.airlift.log.Logging`